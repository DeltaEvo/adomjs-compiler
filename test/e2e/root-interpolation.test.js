const compileAndCreate = require('./compileAndCreate');

const element = compileAndCreate(`{{a}}`);

it('render with a value', () => {
	element.setState({
		a: 'Hello World'
	});
	expect(element.innerHTML).toMatchSnapshot();
});

it('render without value', () => {
	element.setState({
		a: undefined
	});
	expect(element.innerHTML).toMatchSnapshot();
});
