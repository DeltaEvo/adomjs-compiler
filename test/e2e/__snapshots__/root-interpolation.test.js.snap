// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render with a value 1`] = `"Hello World"`;

exports[`render without value 1`] = `"undefined"`;
